README.md

```diff
@@ -48,10 +48,11 @@ steps. The `start-image-build` step initializes local state for the image build,
 and the `finish-image-build` step performs the image building operation with
 Daisy.
 
-Example optional build steps are `run-script` and `install-gpu`. `run-script`
-allows users to customize an image by running a script, and `install-gpu` allows
+Example optional build steps are `run-script`, `install-gpu` and `seal-oem`. `run-script`
+allows users to customize an image by running a script. `install-gpu` allows
 users to install GPU drivers using the
-[COS GPU installer](https://github.com/GoogleCloudPlatform/cos-gpu-installer).
+[COS GPU installer](https://github.com/GoogleCloudPlatform/cos-gpu-installer). 
+`seal-oem` verifies the OEM partition data every time the VM boots.
```

Comment:
Can it be something like -> `seal-oem` allows users to setup a verified OEM partition so that OEM partition data is verified every time the VM boots or a binary from OEM partition is used.